pages/index.js

```diff
@@ -19,7 +19,7 @@ import { normaliseStr } from "../utils/normalize";
 const { publicRuntimeConfig } = getConfig();
 const SITE_DESCRIPTION =
   publicRuntimeConfig.SITE_DESCRIPTION ||
-  "Simplifiez le partage et la circulation des données administratives";
+  "Le site qui simplifie le partage et la circulation des données administratives";
```

Comment:
I'd rather rely on the default value than defining the tagline twice.

Can you remove this ?